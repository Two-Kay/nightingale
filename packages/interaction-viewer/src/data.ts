export type Interaction = {
  accession1: string;
  accession2: string;
  gene?: string;
  interactor1: string;
  interactor2: string;
  organismDiffer?: boolean;
  experiments?: number;
  chain2?: string;
  chain1?: string;
};

export type DbReference = {
  type: string;
  id: string;
};

export type Source = {
  name: string;
  id: string;
  url: string;
  alternativeUrl?: string;
};

export type Evidence = {
  code: string;
  source?: Source;
};

export type Disease = {
  type?: string;
  text?: Text[];
  diseaseId: string;
  acronym?: string;
  dbReference?: DbReference;
  description?: Comment;
};

export type Comment = {
  value: string;
  evidences?: Evidence[];
};

export type Location = {
  location: Comment;
  topology?: Comment;
};

export type SubcellularLocation = {
  type?: string;
  locations: Location[];
  text?: Text[];
  molecule?: string;
};

export type APIInteractionData = {
  accession: string;
  name: string;
  proteinExistence: string;
  taxonomy: number;
  interactions?: Interaction[];
  diseases?: Disease[];
  subcellularLocations?: SubcellularLocation[];
  filterTerms?: string[];
};
