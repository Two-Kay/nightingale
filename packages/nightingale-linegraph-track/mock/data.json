[
  {
    "name": "Chart name",
    "colour": "#FFF .. with default taken from the d3 colour scale if nothing passed?",
    "range": [0, 100],
    "lineCurve": "default linear??",
    "values": [
      {
        "position": 23,
        "value": 45
      }
    ]
  }
]
