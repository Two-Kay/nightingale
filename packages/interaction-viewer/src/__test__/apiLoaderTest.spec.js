import process from "../apiLoader";
import data from "./resources/data";
import diseasesJson from "./resources/diseases.json";

describe("interaction-viewer apiLoader", () => {
  it("should process the data", () => {
    const { adjacencyMap, interactionsMap } = process(data);
    expect(adjacencyMap).toMatchSnapshot();
    expect(interactionsMap).toMatchSnapshot();
  });
});
