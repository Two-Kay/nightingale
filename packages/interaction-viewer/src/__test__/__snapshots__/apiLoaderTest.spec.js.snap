// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interaction-viewer apiLoader should process the data 1`] = `
Array [
  Object {
    "accession": "Q8TD43",
    "interactors": Array [
      "P60903",
      "P31949",
      "P29034",
      "P33764",
      "P26447",
      "P33763",
      "P06703",
      "P05109",
      "P05109",
      "P31949",
      "P29034",
      "P06703",
      "P60903",
      "P33763",
      "P26447",
      "P33764",
    ],
  },
  Object {
    "accession": "P60903",
    "interactors": Array [
      "Q8TD43",
      "Q09666",
      "P07355",
      "P46092",
      "Q14527",
      "Q8NB16-2",
      "P60321",
      "P33764",
      "Q8WXG8",
      "Q8TD43",
      "O75604",
      "Q8N5A5-2",
      "P33764",
    ],
  },
  Object {
    "accession": "P31949",
    "interactors": Array [
      "Q8TD43",
      "Q96HA8",
      "P04271",
      "P04637",
      "Q8TD43",
    ],
  },
  Object {
    "accession": "P29034",
    "interactors": Array [
      "Q8TD43",
      "Q3SXY8",
      "P15311",
      "Q5JX71",
      "Q02790",
      "P52292",
      "Q00987",
      "O15151",
      "P35579",
      "P23297",
      "P29034",
      "P29034",
      "P33764",
      "P04271",
      "P04637",
      "Q8TD43",
      "P52293",
      "P26882",
      "P33764",
    ],
  },
  Object {
    "accession": "P33764",
    "interactors": Array [
      "Q8TD43",
      "P29034",
      "P60903",
      "Q8TBB1",
      "P23297",
      "P60903",
      "P29034",
      "Q8WXG8",
      "P04637",
      "Q8TD43",
    ],
  },
  Object {
    "accession": "P26447",
    "interactors": Array [
      "Q8TD43",
      "P15514",
      "P35070",
      "Q9HB71",
      "P00533",
      "P21860",
      "Q15303",
      "P15311",
      "Q00987",
      "P35579",
      "P23297",
      "P26447",
      "P26447",
      "P04271",
      "P04637",
      "Q8TD43",
    ],
  },
  Object {
    "accession": "P33763",
    "interactors": Array [
      "Q8TD43",
      "Q9HB71",
      "P52907",
      "O15151",
      "P35579",
      "P32418",
      "P04637",
      "Q8TD43",
    ],
  },
  Object {
    "accession": "P06703",
    "interactors": Array [
      "Q8TD43",
      "Q9HB71",
      "O95684",
      "B3EWG5",
      "Q02790",
      "P52292",
      "Q00987",
      "P35579",
      "P50542",
      "P04271",
      "P04637",
      "Q8TD43",
      "P26882",
    ],
  },
  Object {
    "accession": "P05109",
    "interactors": Array [
      "Q8TD43",
      "P04406",
      "P06702",
      "Q8TD43",
    ],
  },
  Object {
    "accession": "Q8TD43-1",
    "interactors": Array [
      "Q8TD43-1",
      "Q8TD43-1",
      "Q8TD43-1",
      "Q8TD43-1",
    ],
  },
  Object {
    "accession": "P04406",
    "interactors": Array [
      "P05109",
    ],
  },
  Object {
    "accession": "P06702",
    "interactors": Array [
      "P05109",
    ],
  },
  Object {
    "accession": "Q96HA8",
    "interactors": Array [
      "P31949",
    ],
  },
  Object {
    "accession": "P04271",
    "interactors": Array [
      "P31949",
      "P29034",
      "P06703",
      "P26447",
    ],
  },
  Object {
    "accession": "P04637",
    "interactors": Array [
      "P31949",
      "P29034",
      "P06703",
      "P33763",
      "P26447",
      "P33764",
    ],
  },
  Object {
    "accession": "Q3SXY8",
    "interactors": Array [
      "P29034",
    ],
  },
  Object {
    "accession": "P15311",
    "interactors": Array [
      "P29034",
      "P26447",
    ],
  },
  Object {
    "accession": "Q5JX71",
    "interactors": Array [
      "P29034",
    ],
  },
  Object {
    "accession": "Q02790",
    "interactors": Array [
      "P29034",
      "P06703",
    ],
  },
  Object {
    "accession": "P52292",
    "interactors": Array [
      "P29034",
      "P06703",
    ],
  },
  Object {
    "accession": "Q00987",
    "interactors": Array [
      "P29034",
      "P06703",
      "P26447",
    ],
  },
  Object {
    "accession": "O15151",
    "interactors": Array [
      "P29034",
      "P33763",
    ],
  },
  Object {
    "accession": "P35579",
    "interactors": Array [
      "P29034",
      "P06703",
      "P33763",
      "P26447",
    ],
  },
  Object {
    "accession": "P23297",
    "interactors": Array [
      "P29034",
      "P26447",
      "P33764",
    ],
  },
  Object {
    "accession": "P52293",
    "interactors": Array [
      "P29034",
    ],
  },
  Object {
    "accession": "P26882",
    "interactors": Array [
      "P29034",
      "P06703",
    ],
  },
  Object {
    "accession": "Q9HB71",
    "interactors": Array [
      "P06703",
      "P33763",
      "P26447",
    ],
  },
  Object {
    "accession": "O95684",
    "interactors": Array [
      "P06703",
    ],
  },
  Object {
    "accession": "B3EWG5",
    "interactors": Array [
      "P06703",
    ],
  },
  Object {
    "accession": "P50542",
    "interactors": Array [
      "P06703",
    ],
  },
  Object {
    "accession": "Q09666",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "P07355",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "P46092",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "Q14527",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "Q8NB16-2",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "P60321",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "Q8WXG8",
    "interactors": Array [
      "P60903",
      "P33764",
    ],
  },
  Object {
    "accession": "O75604",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "Q8N5A5-2",
    "interactors": Array [
      "P60903",
    ],
  },
  Object {
    "accession": "P52907",
    "interactors": Array [
      "P33763",
    ],
  },
  Object {
    "accession": "P32418",
    "interactors": Array [
      "P33763",
    ],
  },
  Object {
    "accession": "P15514",
    "interactors": Array [
      "P26447",
    ],
  },
  Object {
    "accession": "P35070",
    "interactors": Array [
      "P26447",
    ],
  },
  Object {
    "accession": "P00533",
    "interactors": Array [
      "P26447",
    ],
  },
  Object {
    "accession": "P21860",
    "interactors": Array [
      "P26447",
    ],
  },
  Object {
    "accession": "Q15303",
    "interactors": Array [
      "P26447",
    ],
  },
  Object {
    "accession": "Q8TBB1",
    "interactors": Array [
      "P33764",
    ],
  },
]
`;

exports[`interaction-viewer apiLoader should process the data 2`] = `
Map {
  "Q8TD43P60903" => Object {
    "accession1": "Q8TD43",
    "accession2": "P60903",
    "experiments": 2,
    "gene": "S100A10",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-717048",
    "organismDiffer": false,
  },
  "Q8TD43P31949" => Object {
    "accession1": "Q8TD43",
    "accession2": "P31949",
    "experiments": 2,
    "gene": "S100A11",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-701862",
    "organismDiffer": false,
  },
  "Q8TD43P29034" => Object {
    "accession1": "Q8TD43",
    "accession2": "P29034",
    "experiments": 2,
    "gene": "S100A2",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-752230",
    "organismDiffer": false,
  },
  "Q8TD43P33764" => Object {
    "accession1": "Q8TD43",
    "accession2": "P33764",
    "experiments": 2,
    "gene": "S100A3",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-1044747",
    "organismDiffer": false,
  },
  "Q8TD43P26447" => Object {
    "accession1": "Q8TD43",
    "accession2": "P26447",
    "experiments": 2,
    "gene": "S100A4",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-717058",
    "organismDiffer": false,
  },
  "Q8TD43P33763" => Object {
    "accession1": "Q8TD43",
    "accession2": "P33763",
    "experiments": 3,
    "gene": "S100A5",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-7211732",
    "organismDiffer": false,
  },
  "Q8TD43P06703" => Object {
    "accession1": "Q8TD43",
    "accession2": "P06703",
    "experiments": 2,
    "gene": "S100A6",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-352877",
    "organismDiffer": false,
  },
  "Q8TD43P05109" => Object {
    "accession1": "Q8TD43",
    "accession2": "P05109",
    "experiments": 2,
    "gene": "S100A8",
    "interactor1": "EBI-11723041",
    "interactor2": "EBI-355281",
    "organismDiffer": false,
  },
  "Q8TD43-1Q8TD43-1" => Object {
    "accession1": "Q8TD43-1",
    "accession2": "Q8TD43-1",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-20594601",
    "interactor2": "EBI-20594601",
    "organismDiffer": false,
  },
  "P05109P04406" => Object {
    "accession1": "P05109",
    "accession2": "P04406",
    "experiments": 6,
    "gene": "GAPDH",
    "interactor1": "EBI-355281",
    "interactor2": "EBI-354056",
    "organismDiffer": false,
  },
  "P05109P06702" => Object {
    "accession1": "P05109",
    "accession2": "P06702",
    "experiments": 7,
    "gene": "S100A9",
    "interactor1": "EBI-355281",
    "interactor2": "EBI-1055001",
    "organismDiffer": false,
  },
  "P05109Q8TD43" => Object {
    "accession1": "P05109",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-355281",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P31949Q96HA8" => Object {
    "accession1": "P31949",
    "accession2": "Q96HA8",
    "experiments": 3,
    "gene": "NTAQ1",
    "interactor1": "EBI-701862",
    "interactor2": "EBI-741158",
    "organismDiffer": false,
  },
  "P31949P04271" => Object {
    "accession1": "P31949",
    "accession2": "P04271",
    "experiments": 5,
    "gene": "S100B",
    "interactor1": "EBI-701862",
    "interactor2": "EBI-458391",
    "organismDiffer": false,
  },
  "P31949P04637" => Object {
    "accession1": "P31949",
    "accession2": "P04637",
    "experiments": 2,
    "gene": "TP53",
    "interactor1": "EBI-701862",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P31949Q8TD43" => Object {
    "accession1": "P31949",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-701862",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P29034Q3SXY8" => Object {
    "accession1": "P29034",
    "accession2": "Q3SXY8",
    "experiments": 3,
    "gene": "ARL13B",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-11343438",
    "organismDiffer": false,
  },
  "P29034P15311" => Object {
    "accession1": "P29034",
    "accession2": "P15311",
    "experiments": 2,
    "gene": "EZR",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-1056902",
    "organismDiffer": false,
  },
  "P29034Q5JX71" => Object {
    "accession1": "P29034",
    "accession2": "Q5JX71",
    "experiments": 3,
    "gene": "FAM209A",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-18304435",
    "organismDiffer": false,
  },
  "P29034Q02790" => Object {
    "accession1": "P29034",
    "accession2": "Q02790",
    "experiments": 3,
    "gene": "FKBP4",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-1047444",
    "organismDiffer": false,
  },
  "P29034P52292" => Object {
    "accession1": "P29034",
    "accession2": "P52292",
    "experiments": 5,
    "gene": "KPNA2",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-349938",
    "organismDiffer": false,
  },
  "P29034Q00987" => Object {
    "accession1": "P29034",
    "accession2": "Q00987",
    "experiments": 2,
    "gene": "MDM2",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-389668",
    "organismDiffer": false,
  },
  "P29034O15151" => Object {
    "accession1": "P29034",
    "accession2": "O15151",
    "experiments": 2,
    "gene": "MDM4",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-398437",
    "organismDiffer": false,
  },
  "P29034P35579" => Object {
    "accession1": "P29034",
    "accession2": "P35579",
    "experiments": 2,
    "gene": "MYH9",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-350338",
    "organismDiffer": false,
  },
  "P29034P23297" => Object {
    "accession1": "P29034",
    "accession2": "P23297",
    "experiments": 7,
    "gene": "S100A1",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-743686",
    "organismDiffer": false,
  },
  "P29034P29034" => Object {
    "accession1": "P29034",
    "accession2": "P29034",
    "experiments": 6,
    "gene": "S100A2",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-752230",
    "organismDiffer": false,
  },
  "P29034P33764" => Object {
    "accession1": "P29034",
    "accession2": "P33764",
    "experiments": 5,
    "gene": "S100A3",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-1044747",
    "organismDiffer": false,
  },
  "P29034P04271" => Object {
    "accession1": "P29034",
    "accession2": "P04271",
    "experiments": 9,
    "gene": "S100B",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-458391",
    "organismDiffer": false,
  },
  "P29034P04637" => Object {
    "accession1": "P29034",
    "accession2": "P04637",
    "experiments": 4,
    "gene": "TP53",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P29034Q8TD43" => Object {
    "accession1": "P29034",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P29034P52293" => Object {
    "accession1": "P29034",
    "accession2": "P52293",
    "experiments": 2,
    "gene": "Kpna2",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-3043908",
    "organismDiffer": true,
  },
  "P29034P26882" => Object {
    "accession1": "P29034",
    "accession2": "P26882",
    "experiments": 3,
    "gene": "PPID",
    "interactor1": "EBI-752230",
    "interactor2": "EBI-6477155",
    "organismDiffer": true,
  },
  "P06703Q9HB71" => Object {
    "accession1": "P06703",
    "accession2": "Q9HB71",
    "experiments": 2,
    "gene": "CACYBP",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-1047302",
    "organismDiffer": false,
  },
  "P06703O95684" => Object {
    "accession1": "P06703",
    "accession2": "O95684",
    "experiments": 2,
    "gene": "CEP43",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-1266334",
    "organismDiffer": false,
  },
  "P06703B3EWG5" => Object {
    "accession1": "P06703",
    "accession2": "B3EWG5",
    "experiments": 3,
    "gene": "FAM25C",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-14240149",
    "organismDiffer": false,
  },
  "P06703Q02790" => Object {
    "accession1": "P06703",
    "accession2": "Q02790",
    "experiments": 3,
    "gene": "FKBP4",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-1047444",
    "organismDiffer": false,
  },
  "P06703P52292" => Object {
    "accession1": "P06703",
    "accession2": "P52292",
    "experiments": 3,
    "gene": "KPNA2",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-349938",
    "organismDiffer": false,
  },
  "P06703Q00987" => Object {
    "accession1": "P06703",
    "accession2": "Q00987",
    "experiments": 2,
    "gene": "MDM2",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-389668",
    "organismDiffer": false,
  },
  "P06703P35579" => Object {
    "accession1": "P06703",
    "accession2": "P35579",
    "experiments": 2,
    "gene": "MYH9",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-350338",
    "organismDiffer": false,
  },
  "P06703P50542" => Object {
    "accession1": "P06703",
    "accession2": "P50542",
    "experiments": 3,
    "gene": "PEX5",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-597835",
    "organismDiffer": false,
  },
  "P06703P04271" => Object {
    "accession1": "P06703",
    "accession2": "P04271",
    "experiments": 6,
    "gene": "S100B",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-458391",
    "organismDiffer": false,
  },
  "P06703P04637" => Object {
    "accession1": "P06703",
    "accession2": "P04637",
    "experiments": 3,
    "gene": "TP53",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P06703Q8TD43" => Object {
    "accession1": "P06703",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P06703P26882" => Object {
    "accession1": "P06703",
    "accession2": "P26882",
    "experiments": 3,
    "gene": "PPID",
    "interactor1": "EBI-352877",
    "interactor2": "EBI-6477155",
    "organismDiffer": true,
  },
  "P60903Q09666" => Object {
    "accession1": "P60903",
    "accession2": "Q09666",
    "experiments": 2,
    "gene": "AHNAK",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-2555881",
    "organismDiffer": false,
  },
  "P60903P07355" => Object {
    "accession1": "P60903",
    "accession2": "P07355",
    "experiments": 4,
    "gene": "ANXA2",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-352622",
    "organismDiffer": false,
  },
  "P60903P46092" => Object {
    "accession1": "P60903",
    "accession2": "P46092",
    "experiments": 5,
    "gene": "CCR10",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-348022",
    "organismDiffer": false,
  },
  "P60903Q14527" => Object {
    "accession1": "P60903",
    "accession2": "Q14527",
    "experiments": 2,
    "gene": "HLTF",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-1045161",
    "organismDiffer": false,
  },
  "P60903Q8NB16-2" => Object {
    "accession1": "P60903",
    "accession2": "Q8NB16-2",
    "experiments": 3,
    "gene": "MLKL",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-19046912",
    "organismDiffer": false,
  },
  "P60903P60321" => Object {
    "accession1": "P60903",
    "accession2": "P60321",
    "experiments": 3,
    "gene": "NANOS2",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-10216569",
    "organismDiffer": false,
  },
  "P60903P33764" => Object {
    "accession1": "P60903",
    "accession2": "P33764",
    "experiments": 3,
    "gene": "S100A3",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-1044747",
    "organismDiffer": false,
  },
  "P60903Q8WXG8" => Object {
    "accession1": "P60903",
    "accession2": "Q8WXG8",
    "experiments": 6,
    "gene": "S100Z",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-12198403",
    "organismDiffer": false,
  },
  "P60903Q8TD43" => Object {
    "accession1": "P60903",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P60903O75604" => Object {
    "accession1": "P60903",
    "accession2": "O75604",
    "experiments": 3,
    "gene": "USP2",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-743272",
    "organismDiffer": false,
  },
  "P60903Q8N5A5-2" => Object {
    "accession1": "P60903",
    "accession2": "Q8N5A5-2",
    "experiments": 3,
    "gene": "ZGPAT",
    "interactor1": "EBI-717048",
    "interactor2": "EBI-10183064",
    "organismDiffer": false,
  },
  "P33763Q9HB71" => Object {
    "accession1": "P33763",
    "accession2": "Q9HB71",
    "experiments": 3,
    "gene": "CACYBP",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-1047302",
    "organismDiffer": false,
  },
  "P33763P52907" => Object {
    "accession1": "P33763",
    "accession2": "P52907",
    "experiments": 2,
    "gene": "CAPZA1",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-355586",
    "organismDiffer": false,
  },
  "P33763O15151" => Object {
    "accession1": "P33763",
    "accession2": "O15151",
    "experiments": 2,
    "gene": "MDM4",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-398437",
    "organismDiffer": false,
  },
  "P33763P35579" => Object {
    "accession1": "P33763",
    "accession2": "P35579",
    "experiments": 2,
    "gene": "MYH9",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-350338",
    "organismDiffer": false,
  },
  "P33763P32418" => Object {
    "accession1": "P33763",
    "accession2": "P32418",
    "experiments": 2,
    "gene": "SLC8A1",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-2682189",
    "organismDiffer": false,
  },
  "P33763P04637" => Object {
    "accession1": "P33763",
    "accession2": "P04637",
    "experiments": 2,
    "gene": "TP53",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P33763Q8TD43" => Object {
    "accession1": "P33763",
    "accession2": "Q8TD43",
    "experiments": 3,
    "gene": "TRPM4",
    "interactor1": "EBI-7211732",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P26447P15514" => Object {
    "accession1": "P26447",
    "accession2": "P15514",
    "experiments": 5,
    "gene": "AREG",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-953674",
    "organismDiffer": false,
  },
  "P26447P35070" => Object {
    "accession1": "P26447",
    "accession2": "P35070",
    "experiments": 2,
    "gene": "BTC",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-6590057",
    "organismDiffer": false,
  },
  "P26447Q9HB71" => Object {
    "accession1": "P26447",
    "accession2": "Q9HB71",
    "experiments": 2,
    "gene": "CACYBP",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-1047302",
    "organismDiffer": false,
  },
  "P26447P00533" => Object {
    "accession1": "P26447",
    "accession2": "P00533",
    "experiments": 6,
    "gene": "EGFR",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-297353",
    "organismDiffer": false,
  },
  "P26447P21860" => Object {
    "accession1": "P26447",
    "accession2": "P21860",
    "experiments": 4,
    "gene": "ERBB3",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-720706",
    "organismDiffer": false,
  },
  "P26447Q15303" => Object {
    "accession1": "P26447",
    "accession2": "Q15303",
    "experiments": 4,
    "gene": "ERBB4",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-80371",
    "organismDiffer": false,
  },
  "P26447P15311" => Object {
    "accession1": "P26447",
    "accession2": "P15311",
    "experiments": 3,
    "gene": "EZR",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-1056902",
    "organismDiffer": false,
  },
  "P26447Q00987" => Object {
    "accession1": "P26447",
    "accession2": "Q00987",
    "experiments": 3,
    "gene": "MDM2",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-389668",
    "organismDiffer": false,
  },
  "P26447P35579" => Object {
    "accession1": "P26447",
    "accession2": "P35579",
    "experiments": 18,
    "gene": "MYH9",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-350338",
    "organismDiffer": false,
  },
  "P26447P23297" => Object {
    "accession1": "P26447",
    "accession2": "P23297",
    "experiments": 4,
    "gene": "S100A1",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-743686",
    "organismDiffer": false,
  },
  "P26447P26447" => Object {
    "accession1": "P26447",
    "accession2": "P26447",
    "experiments": 6,
    "gene": "S100A4",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-717058",
    "organismDiffer": false,
  },
  "P26447P04271" => Object {
    "accession1": "P26447",
    "accession2": "P04271",
    "experiments": 8,
    "gene": "S100B",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-458391",
    "organismDiffer": false,
  },
  "P26447P04637" => Object {
    "accession1": "P26447",
    "accession2": "P04637",
    "experiments": 9,
    "gene": "TP53",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P26447Q8TD43" => Object {
    "accession1": "P26447",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-717058",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
  "P33764Q8TBB1" => Object {
    "accession1": "P33764",
    "accession2": "Q8TBB1",
    "experiments": 3,
    "gene": "LNX1",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-739832",
    "organismDiffer": false,
  },
  "P33764P23297" => Object {
    "accession1": "P33764",
    "accession2": "P23297",
    "experiments": 3,
    "gene": "S100A1",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-743686",
    "organismDiffer": false,
  },
  "P33764P60903" => Object {
    "accession1": "P33764",
    "accession2": "P60903",
    "experiments": 3,
    "gene": "S100A10",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-717048",
    "organismDiffer": false,
  },
  "P33764P29034" => Object {
    "accession1": "P33764",
    "accession2": "P29034",
    "experiments": 5,
    "gene": "S100A2",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-752230",
    "organismDiffer": false,
  },
  "P33764Q8WXG8" => Object {
    "accession1": "P33764",
    "accession2": "Q8WXG8",
    "experiments": 3,
    "gene": "S100Z",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-12198403",
    "organismDiffer": false,
  },
  "P33764P04637" => Object {
    "accession1": "P33764",
    "accession2": "P04637",
    "experiments": 2,
    "gene": "TP53",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-366083",
    "organismDiffer": false,
  },
  "P33764Q8TD43" => Object {
    "accession1": "P33764",
    "accession2": "Q8TD43",
    "experiments": 2,
    "gene": "TRPM4",
    "interactor1": "EBI-1044747",
    "interactor2": "EBI-11723041",
    "organismDiffer": false,
  },
}
`;
